src/main/java/com/nike/cerberus/operation/core/CreateLoadBalancerOperation.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2016 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.operation.core;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nike.cerberus.ConfigConstants;
+import com.nike.cerberus.command.core.CreateLoadBalancerCommand;
+import com.nike.cerberus.domain.EnvironmentMetadata;
+import com.nike.cerberus.domain.cloudformation.LoadBalancerParameters;
+import com.nike.cerberus.domain.cloudformation.VpcOutputs;
+import com.nike.cerberus.domain.environment.StackName;
+import com.nike.cerberus.operation.Operation;
+import com.nike.cerberus.service.CloudFormationService;
+import com.nike.cerberus.store.ConfigStore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.util.Map;
+
+import static com.nike.cerberus.module.CerberusModule.CF_OBJECT_MAPPER;
+
+/**
+ * Creates the base components via CloudFormation used by all of Cerberus.
+ */
+public class CreateLoadBalancerOperation implements Operation<CreateLoadBalancerCommand> {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private final EnvironmentMetadata environmentMetadata;
+
+    private final CloudFormationService cloudFormationService;
+
+    private final ConfigStore configStore;
+
+    private final ObjectMapper cloudFormationObjectMapper;
+
+    @Inject
+    public CreateLoadBalancerOperation(final EnvironmentMetadata environmentMetadata,
+                              final CloudFormationService cloudFormationService,
+                              final ConfigStore configStore,
+                              @Named(CF_OBJECT_MAPPER) final ObjectMapper cloudformationObjectMapper) {
+        this.environmentMetadata = environmentMetadata;
+        this.cloudFormationService = cloudFormationService;
+        this.configStore = configStore;
+        this.cloudFormationObjectMapper = cloudformationObjectMapper;
+    }
+
+    @Override
+    public void run(final CreateLoadBalancerCommand command) {
+        final String environmentName = environmentMetadata.getName();
+        final VpcOutputs vpcOutputs = configStore.getVpcStackOutputs();
+
+        final String sslCertificateArn = configStore.getServerCertificateArn(StackName.CMS)
```

Comment:
This variable is not used. is that a mistake?

Reply:
@fieldju This one is used in line 75. Was there another one you saw maybe?

Reply:
no, my IDE was being dumb